### PR TITLE
Apply environment overrides to a LBFargateManifest configuration

### LINKED ISSUE #252: Apply environment overrides from app manifest to CFN template
## Task
The CFN template generated from an application's manifest right now ignores environment overrides (https://github.com/aws/amazon-ecs-cli-v2/issues/251).   
We want to apply any field overwritten for the environment we want the application deployed to.  
So that the appropriate parameter values are generated in the CFN template.

### Related to
Parent story for this task: https://github.com/aws/amazon-ecs-cli-v2/issues/229

### PR DESCRIPTION
**Pull request by efekarakus**
Add a new method `EnvConf` that returns the manifest's configuration for a given environment.

Resolves #252

_By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice._

### CODE REVIEW

File: internal/pkg/manifest/lb_fargate_manifest.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ type ImageWithPort struct {
 type LBFargateConfig struct {
 	RoutingRule      `yaml:"http,flow"`
 	ContainersConfig `yaml:",inline"`
-	Public           bool               `yaml:"public"`
+	Public           *bool              `yaml:"public"` // A pointer because we don't want the environment override to default to false.
```
Comment (kohidave) [inline internal/pkg/manifest/lb_fargate_manifest.go]: I don't know if we even need this parameter.

Comment (efekarakus) [reply internal/pkg/manifest/lb_fargate_manifest.go]: me neither what would we expect if a user set this to false?

Comment (SoManyHs) [reply internal/pkg/manifest/lb_fargate_manifest.go]: hmm....that is confusing. Does this mean it wouldn't be a public Load Balancer, or that the task itself is not public?

Comment (efekarakus) [reply internal/pkg/manifest/lb_fargate_manifest.go]: Removed the field!   
We can add it back later if we can come up with a use-case, right now we have a public loadbalancer and the service will receive requests from it.

**Pull request merged**